src/system.h

```diff
@@ -441,6 +441,12 @@ struct System {
         }
     }
 
+    void UpdateAmountStatus(Selection &s) {
+        if (frame->GetStatusBar()) {
+            frame->SetStatusText(wxString::Format(_(L"Amount %d"), s.xs * s.ys), 4);

```

Comment:
maybe `%d cells` is more informative?

Reply:
That's right! I updated the PR according to your proposal.